lib/src/benchmark_base.dart

```diff
@@ -39,21 +39,17 @@ class BenchmarkBase {
     int elapsed = 0;
     Stopwatch watch = Stopwatch();
     watch.start();
-    while (iter > 0) {
+    while (true) { // ignore: literal_only_boolean_expressions
```

Comment:
Nit: This is a cleaner way to express this without having to ignore a warning:

```dart
for (;;) {
  ...
}
```

Reply:
Or drop the lint, since it's clearly unreasonable here. :)